action.yml

```diff
@@ -182,7 +182,7 @@ runs:
       with:
         issue-number: ${{ github.event.pull_request.number }}
         comment-author: 'github-actions[bot]'
-        body-includes: "Benchmark Results (Julia v${{ inputs.julia-version }})"
+        body-includes: "Benchmark Results (Julia ${{ inputs.julia-version }})"
```

Comment:
Tiny thing: I think this means if the action is updated between first benchmark (when the comment is first posted) and any later benchmark on the same PR, new comments would get generated by the bot, as it wouldn't match the original.

This is a one-time cost. But I do have some PRs that have been open for months, and those PRs would (I think) get new comments from the actions bot.